src/PKSim.Infrastructure/ORM/Repositories/PopulationAgeRepository.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using OSPSuite.Utility.Collections;
+using PKSim.Core;
+using PKSim.Core.Model;
+using PKSim.Core.Repositories;
+using PKSim.Infrastructure.ORM.Core;
+using PKSim.Infrastructure.ORM.Mappers;
+
+namespace PKSim.Infrastructure.ORM.Repositories
+{
+   public interface IFlatPopulationAgeRepository : IMetaDataRepository<PopulationAgeSettings>
+   {
+   }
+
+   public class FlatPopulationAgeRepository : MetaDataRepository<PopulationAgeSettings>, IFlatPopulationAgeRepository
+   {
+      public FlatPopulationAgeRepository(IDbGateway dbGateway,
+         IDataTableToMetaDataMapper<PopulationAgeSettings> mapper)
+         : base(dbGateway, mapper, CoreConstants.ORM.ViewPopulationAge)
+      {
+      }
+   }
+
+   public class PopulationAgeRepository : StartableRepository<PopulationAgeSettings>, IPopulationAgeRepository
+   {
+      private readonly IFlatPopulationAgeRepository _flatPopulationAgeRepository;
+      private readonly Cache<string, PopulationAgeSettings> _populationAgeSettings = new Cache<string, PopulationAgeSettings>();
+
+      public PopulationAgeRepository(IFlatPopulationAgeRepository flatPopulationAgeRepository)
+      {
+         _flatPopulationAgeRepository = flatPopulationAgeRepository;
+      }
+
+      public override IEnumerable<PopulationAgeSettings> All()
+      {
+         Start();
```

Comment:
Start was missing. That's why in the tests, the issue was being seen (we do not start all repo in the test, only what's required)